projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/index.php

```diff
@@ -637,6 +637,19 @@ function wpcom_site_has_global_styles_feature( $blog_id = 0 ) {
 		}
 	}
 
+	// If the GLOBAL_STYLES_ON_PERSONAL_PLAN feature is enabled, we need to check if the site has a Personal plan and add the sticker.
```

Comment:
I think this needs to be generic so we can reuse it in the logic for custom CSS addon; I think the sticker should be re-used there.

Reply:
Let's not fall into early generalization. Especially considering there is a 99.99% chance of not reverting. We have an issue for handling the Custom CSS add-on ([here](https://github.com/Automattic/dotcom-forge/issues/9519))

Reply:
Yeah, I'm working on that issue and realized I needed something similar for the custom CSS too. :) 

I'll copy your code so that we don't block this one and do a refactor after these two implementations land. Sounds good? :)

Reply:
I doubt we need a refactor, bear in mind we're going to delete this code eventually.